aspnetcore/toc.yml

```
@@ -614,6 +614,8 @@
       uid: test/loadtests
     - name: Troubleshoot
       uid: test/troubleshoot
+    - name: Debug ASP.NET Core apps
```

```suggestion
    - name: Debug on Unix over SSH
```

🤔 I think "ASP.NET Core" can be assumed.